stake-pool/program/src/state.rs

```diff
@@ -67,6 +67,8 @@ pub enum State {
 }
 
 impl State {
+    /// Length of state data when serialized
+    pub const LEN: usize = size_of::<u8>() + size_of::<StakePool>();
```

Comment:
Why is the extra `u8` needed here?

Reply:
Please check several lines below: the first byte is used to encode account status (initialized or not) and the rest is used for the stake pool status struct.

Reply:
Ah gotcha, I was only looking at the changed code.  Thanks for the clarification